### ISSUE TITLE
**Advanced Guides** section not completely visible when opened

### ISSUE THREAD

**Issue by Legends** [reactjs/reactjs.org#1515]
**Do you want to request a *feature* or report a *bug*?**
Bug in your website - Chrome Version 71.0.3578.98 (Official Build) (64-bit)

**What is the current behavior?**
Cannot see the whole [sidebar](https://reactjs.org/docs/refs-and-the-dom.html) when **Advanced Guides** section is open.

![image](https://user-images.githubusercontent.com/966368/50541603-83324880-0ba9-11e9-949d-8786ac048e20.png)

If I zoom out to about 60% I can see the whole sidebar!
The same in Edge, FF.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem:**
Click the last visible entry in the sidebar. you can also scroll to the bottom of the content, but the rest of the sidebar stays invisible.

**What is the expected behavior?**
 See whole sidebar

**Comment by alexkrolick** [comment]
You should be able to scroll the sidebar independently from the main page - the scroll bar for it is hidden, though.

**Comment by Legends** [comment]
Can't see any scrollbar unfortunately...

**Comment by alexkrolick** [comment]
Can you scroll with a mouse wheel or the up/down arrow keys while hovering over the sidebar?

Flagging this as an accessibility issue.

**Comment by Shaw007134** [comment]
> I cannot scroll using the up/down arrow keys while hovering.
> 
> But the following works in my case:
> 
> When I click on a menu entry like "Refs and the DOM" then the main content gets loaded and gets the focus.
> Now when I click on the same menu entry "Refs and the DOM" again, the sidebar gets the focus and now I am able to scroll up or down using the arrows keys, but still no scrollbar.

I have exactly the same issue with Chrome V71, the sidebar cannot be scrolled up or down, and cannot see the full advanced guides,

**Comment by Marcholio** [comment]
Works on Chrome V72.0 on Linux.

**Comment by Legends** [comment]
Just updated to v72. I still have to click twice on the same sidebar entry in order to stay on the same page by being able to scroll the sidebar.
Just hovering over it, still doesn't work.

**Comment by InteriorNature** [comment]
It appears that the -webkit-overflow-scrolling: touch CSS property is non-standard, and both Chrome and Mozilla browsers will not use it when at 100% in a windows full-screen environment (and maybe other OS.) So I have no problem accessing the full sidebar on my phone, but a scroll option does not come up on my computer. I'm shrinking my views down 75% to access additional docs.

**Comment by chiangs** [comment]
Is this issue still open? I don't seem to get the same problems in chrome 77.0.3865.90 nor FF DE 70.0b8

**Comment by Legends** [comment]
Just checked it, now it seems to work. Same Chrome version.
